server/users.js

```diff
@@ -993,14 +1010,15 @@ class User extends Chat.MessageContext {
 	 * @param {string[]} updated the settings which have been updated or none for all settings.
 	 */
 	getUpdateuserText(...updated) {
+		const status = this.status ? `@${this.status}` : '';
 		const named = this.named ? 1 : 0;
 		const diff = {};
 		const settings = updated.length ? updated : SETTINGS;
 		for (const setting of settings) {
 			// @ts-ignore - dynamic lookup
 			diff[setting] = this[setting];
 		}
-		return `|updateuser|${this.name}|${named}|${this.avatar}|${JSON.stringify(diff)}`;
+		return `|updateuser|${this.name}${status}|${named}|${this.avatar}|${JSON.stringify(diff)}`;
```

Comment:
I can't imagine a single client/bot actually choking on being sent `getIdentity` here. Especially if it can handle the `@status` part of the change.

Reply:
```suggestion
		return `|updateuser|${this.getIdentityWithStatus()}|${named}|${this.avatar}|${JSON.stringify(diff)}`;
```

Reply:
> I can't imagine a single client/bot actually choking on being sent `getIdentity` here. 

https://github.com/Zarel/Pokemon-Showdown-Client/blob/master/src/battle-text-parser.ts#L60-L74

100% breaks. Original author was garbage though, TBF.

Reply:
Does it? It adds the symbol to your name but in theory that shouldn't cause any crashes/bugs.

Reply:
Is displaying a moderator's name as ``@@pre`` not a bug? (or maybe it won't do that, I should really spin up a server and test)

Reply:
I wouldn't call that "choking"; choking is a bug that makes it impossible to use. I'm also not sure if that would be the actual consequence here.

Reply:
My guess is that it would show your symbol in the upper right but there wouldn't be any other consequences.

Reply:
Im going to verify now how the official client would behave.

However, not to mince words - choking/bug/'feature' what have you, I think its not unreasonable to expect some clients might be confused with having a new element (group rank symbol) in the same place there was just a name. If we document that in our protocol then sure (and I'll update the client code to handle that as well, so theres no weirdness/display issues), but I think adding it currently would not be ideal/cause small display issues.

Reply:
The reason I'm not worried is because names can contain symbols. A client would have to hardcode names not containing symbols to seriously choke on the protocol change.

And it's a bit irrelevant because we're already making a bigger change that's much more likely to break other clients, anyway.